dotcom-rendering/src/model/enhanceCards.ts

```diff
@@ -109,5 +109,11 @@ export const enhanceCards = (
 					  )
 					: undefined,
 				discussionId: faciaCard.discussion.discussionId,
+				// nb. there is a distinct 'byline' property on FEFrontCard, at
```

Comment:
As noted here, there are two `byline` properties on an FEFrontCard. I've preferred this one, even though it's more deeply nested, because I'm not aware of any differences between the two, and because in general it seems as though we get card contents from inside `maybeContent`.

Does anyone know of any scenarios in which the two `byline` values might be different?

Reply:
The second property looks to be optional whereas the first is (currently) required so the first might be cleaner? I think to really understand the differences we'd need to dig into Frontend.

Reply:
Where is it defined as required? I was basing this on the the type definition for `FEFrontCard` here, where it looks like they're both optional, so maybe we need to update that? https://github.com/guardian/dotcom-rendering/blob/2dd43453e2d425dc652443f1ee743315e58887ec/dotcom-rendering/index.d.ts#L768-L791

Reply:
Oh interesting, they're both optional. Well, then in that case, I think you're previous reasoning sounds good - lets favour this part of the object from where we read other properties